upload/src/addons/SV/WarningImprovements/Setup.php

```diff
@@ -48,7 +48,7 @@ public function installStep3()
     {
         $db = $this->db();
 
-        $defaultRegisteredGroupId = 2;
+        $defaultRegisteredGroupId = \XF\Entity\User::GROUP_REG;
```

Comment:
@Xon Or is the add-on supposed to be hard coded with default registered group id as 2?

Reply:
That is the default, I couldn't actually find where that constant had moved to! from XF1 => XF2